### DISCUSSION TITLE
Installing with httpd server already running

### DISCUSSION THREAD

**Discussion by nicrame** [dec0dOS/zero-ui#141] [Q&A]
Hello.
I want to try zero-ui but couldn't start it.
My enviroment is CentOS 8, with httpd already running (but only on 3 IP addresses, 2 are not used).
At first there was this error (after docker-compose up -d --no-build command):
```
Starting zu-https-proxy  ... error

ERROR: for zu-https-proxy  Cannot start service https-proxy: driver failed programming external connectivity on endpoint zu-https-proxy (62e2df98e4efde82adfa63bed439f28927b9e3bf2ab77bf6e175b140784c4d67): Error starting userland proxy: listen tcp4 0.0.0.0:443: bind: address already in use

ERROR: for https-proxy  Cannot start service https-proxy: driver failed programming external connectivity on endpoint zu-https-proxy (62e2df98e4efde82adfa63bed439f28927b9e3bf2ab77bf6e175b140784c4d67): Error starting userland proxy: listen tcp4 0.0.0.0:443: bind: address already in use
ERROR: Encountered errors while bringing up the project.
```
So i tried to setup one speicific IP address that will be used by editing docker-compose.yml and adding "- LISTEN_ADDRESS=my.ip.add.res" inside "environment:" (but i'm not sure if it is correct place for that).
But that didn't change anything.
So i removed whole https-proxy section from docker-compose.yml, and there is also instruction that says "...and change zero-ui port expose to ports.". But i do not understand it. There is already "ports" like:
```
    expose:
      - "9993/tcp"
    ports:
      - "9993:9993/udp"
```
So should i put "9993:9993/udp" inside expose?
Can anyone help please?

**Comment by ThierryBegin** [comment]
They are refeering to the 
```zero-ui:
    image: dec0dos/zero-ui:latest
    container_name: zu-main
    build:
      context: ...
 ```
part in the docker-compose.
You need to add the port 4000
```    
   expose:
      - "4000"
   ports:
      - "4000:4000"
```

and then with your own proxy you'll be able to connect it through port 4000
